### ISSUE TITLE
Better gender balance for the audience

### ISSUE THREAD

**Issue by bontoJR** [swissmobidevs/appbuilders16#15]
As pointed out, gender is extremely unbalanced in the audience. We tried to do our best with speakers which were a 70-30%, still unfair from a reasonable perspective point of view, but how can this be improved? Any idea?

**Comment by philhinco** [comment]
Even though its true that the audience was mostly male, that is kind of an illness of our profession. The majority of developers in software engineering are male:
[LinkedIn: 16% female](https://business.linkedin.com/talent-solutions/blog/2014/03/women-in-engineering-the-sobering-stats)
[StackOverflow Dev Survey 2015: 5.8(!!)% female](http://stackoverflow.com/research/developer-survey-2015#profile-gender)

Maybe reaching out to Women In Tech groups online could yield more women sending in CfPs, or actively approaching female giants in our industry.

You should just try to do your best to balance the speakers, but also keep the end goal of having the best talks possible in mind, so if you only reach 70/30 or 65/35 but all the talks are great then that's all good.

Balancing the audience is kinda tricky, if even possible.
I don't really have any ideas here, maybe someone else?

**Comment by bontoJR** [comment]
@philhinco thanks for the amazing insight, I never had the chance to get numbers about the male/female status of the industry. 

I had a private discussion with a good friend which suggested to involve more groups, like you suggested and also see if some company can sponsorize some tickets to have them more affordable. I don't know if the Swiss industry is open to this kind of support, I honestly have to explore this space.

**Comment by philhinco** [comment]
@bontoJR Yeah, making tickets more affordable would be an idea. Or maybe allocating a batch of tickets specifically to for Women in Tech groups.

The easiest way to find that out is to approach them. Maybe some sponsor packs could specifically involve subsidizing tickets for women or other minorities in exchange for some kind of benefit at the conference.
One thing I wouldn't do is specifically saying "half the tickets for men, half for women". That's probably an unattainable goal to have an even split among gender lines, especially with the numbers we have.

**Comment by bontoJR** [comment]
>  One thing I wouldn't do is specifically saying "half the tickets for men, half for women". That's probably an unattainable goal to have an even split among gender lines, especially with the numbers we have.

I totally agree with that, I am not aiming to 50-50 which is an unrealistic goal, but I think we can do a little better for sure. 👍 

Thank you very much for your opinion and the data, I really appreciated that!

**Comment by philhinco** [comment]
No problem!

It's always possible to improve, maybe next year we can get to 65/35 or even 60/40.

**Comment by akosma** [comment]
Make it explicit in the CfP request, in your website, in your promotional material, that you are "prioritising talks from underrepresented groups in the industry" which includes not only women, but many other groups that must be given a bigger and better chance. Being very clear about this objective from a PR point of view will help you achieve greater diversity.

**Comment by akosma** [comment]
I want to stress this fact: this should not only be about gender. It is very important that we bring more women, and for a country that has such diversity as Switzerland (35% foreign nationals) it would be great to have attendees and speakers from all continents, skin colors, origins, languages, genders, and any other variable that I can think of.

**Comment by akosma** [comment]
As @philhinco said, beware of language: some countries (some Scandinavian nations, for example) explicitly forbid gender privilege texts, so it is important to phrase it in an open but neutral way. Hard balance to achieve but totally worth it.

**Comment by neonichu** [comment]
Just dropping this post from the eurucamp organisers here http://blog.eurucamp.org/2015/08/12/accessibility-diversity/ which has a lot of good info on how they achieved a very diverse speaker lineup and attendance.

**Comment by bontoJR** [comment]
Thanks @neonichu, amazing article!
